roles/tests/test_user_models.py

```diff
@@ -51,3 +51,11 @@ def __init__(self, foo_var):
 
         user_group2 = FooObject('bar')
         self.assertFalse(user_group1.__eq__(user_group2))
+
+    def test_user_group_hash(self):
+        """Test for UserGroup __hash__"""
+        user_group = UserGroup('foo1', 'bar')
+        self.assertEqual(
+            user_group.__hash__(),
+            hash((user_group.username, user_group.group_type))
```

Comment:
I think it would be nicer if the test created another `UserGroup` instance (even if it wasn't saved to the database) and called `hash()` on both and compared the results. And maybe make a one-character change to `username` and `group_type` and prove `hash()` returns different results.